web/services/getUserCustomersService.php

```diff
@@ -31,23 +31,17 @@
     include_once(PHPREPORT_ROOT . '/model/facade/CustomersFacade.php');
     include_once(PHPREPORT_ROOT . '/model/vo/UserVO.php');
 
-    $userLogin = $_GET['uid'];
-
-    $active = $_GET['active'];
+    $userLogin = $_GET['uid'] ?? "";
 
+    $active = $_GET['active'] ?? "";
     if (strtolower($active) == "true")
-    $active = True;
+        $active = True;
     else
-    $active = False;
-
-    $login = $_GET['login'];
-
-    $sid = $_GET['sid'];
+        $active = False;
```

Comment:
This is nitpick, but I like removing the Else when possible for a cleaner code, like:

```
    $active = False;
    if (strtolower($active) == "true")
        $active = True;
```